packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -122,8 +138,29 @@ function ChangelogTooltip<TRef extends HTMLElement>({
               >
                 {postDateFormat(post.createdAt)}
               </time>
-              <div className="mt-2 w text-theme-label-tertiary typo-callout">
-                {post.summary}
+              {!!post.summary && (
+                <div
+                  className="mt-2.5 w text-theme-label-tertiary typo-callout"
+                  data-testid="changelogSummary"
+                >
+                  {post.summary}
+                </div>
+              )}
+              <div className="flex gap-4 py-1.5 px-2.5 mt-4 font-bold w text-theme-color-salt typo-footnote">
+                <span
+                  className="flex gap-1.5"
+                  data-testid="changelogUpvotesCounter"
+                >
+                  <UpvoteIcon secondary />
+                  <InteractionCounter value={post.numUpvotes} />
+                </span>
+                <span
+                  className="flex gap-1.5"
+                  data-testid="changelogCommentsCounter"
+                >
+                  <CommentIcon secondary />
+                  <InteractionCounter value={post.numComments} />
+                </span>
```

Comment:
We should avoid using the `.5` values in our tailwindcss as our spacing should only be divisible by 4. When these cases arise, the design team mentioned to tag them in Slack for visibility and to be able for them to correct it.